README.md

```diff
@@ -1,2 +1,150 @@
-# portfolio-setup-and-mobile-first
-my personal portfolio's first sketches
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="images\testlogo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>ReadMe</b></h3>
+
+</div>
+
+### 👤 Author
+  
+**Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Personal Portfolio <a name="about-project"></a>
```

Comment:
- [ ] Kindly consider adding a description for your project here. it will make your readme more professional. Also, we are encouraged by Microverse to not delete any section of the readme and to fill every section with appropriate information.